mkdocs.yml

```diff
@@ -183,7 +183,9 @@ nav:
           - Pulse / Ivanti Secure Connect: xdr/features/collect/integrations/network/pulse.md
           - Rubycat PROVE IT: xdr/features/collect/integrations/network/rubycat_prove_it.md
           - SonicWall Firewall: xdr/features/collect/integrations/network/sonicwall_fw.md
-          - Sophos Firewall: xdr/features/collect/integrations/network/sophos_fw.md
+          - Sophos:
+            - Sophos Firewall: xdr/features/collect/integrations/network/sophos_fw.md
+            - Sophos Analysis Threat Center: xdr/features/collect/integrations/network/sophos-analysis-threat-center.md
```

Comment:
Could you rename the title to `Sophos Threat Analysis Center` and the filename to `sophos-threat-analysis-center.md`?
Also, I think that the section `Cloud and Saas` is a more suitable place for this integration.